src/lib/createPollBlocks.ts

```diff
@@ -141,7 +152,8 @@ export function createPollBlocks(block: BlockBuilder, question: string, options:
 
     block.addContextBlock({
         elements: [
-            block.newMarkdownTextObject(`${ poll.totalVotes } votes ${ poll.finished ? '| Final Results' : '' }`),
+            block.newMarkdownTextObject(`*${poll.totalVotes}* total votes *${getUniqueVoters(poll)}* users voted ${poll.finished ? '| Final Results' : ''}`),
```

Comment:
```suggestion
            block.newMarkdownTextObject(`*${poll.totalVotes}* total votes | *${getUniqueVoters(poll)}* users voted ${poll.finished ? '| Final Results' : ''}`),
```
It would be good to have some kind of visual separation between the two